locale/en.po

```diff
@@ -3070,11 +3070,19 @@ msgstr "Overwrite existing save:"
 msgid "OVERWRITE_EXISTING_SAVE_TITLE"
 msgstr "Overwrite existing save?"
 
-#: ../src/saving/SaveHelper.cs:379
+#: ../src/saving/SaveHelper.cs:382
+msgid "SAVING_NOT_POSSIBLE"
+msgstr "Saving is not currently possible due to:"
```

Comment:
```suggestion
msgstr "Saving is currently not possible due to:"
```

Reply:
I found this: https://english.stackexchange.com/questions/5464/currently-not-or-not-currently
which says that both ways are correct. Though, having "not" first emphasises the "not", which I don't think is too much of a difference either way, so... I'm not convinced it's worth the minuscule trouble of changing this.

Reply:
Would you consider that a fine explanation? If you can get people to reply on discord in the #translation channel or maybe on the community forums, this'll be pretty easy to change on https://translate.revolutionarygamesstudio.com/

Reply:
Nah, should be fine. I've talked with somebody who is better in english than I am, and he said that both are acceptable. No need to change it.

Reply:
Okay that's fine. But as I linked that stackexchange answer it says that there is a slight emphasis difference, which we can discuss in the future if anyone cares enough to want to change it.